app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AmusementParksController < ApplicationController
+  def show
+    @amusement_park = AmusementPark.find(params[:id])
+    @mechanics = @amusement_park.mechanics_list
+  end
+end
```

Comment:
In your reflection, you mentioned not seeing the point of not just doing the AR in the controller since it seems a little bit much to move that logic to the model only to call on the method here. This advice comes from an old coding paradigm called "Fat Model and Skinny Controllers" ... there are differing opinions about following this principle. 

The idea behind this is that since we are dealing with data retrieval and/or performing operations on data, we want to encapsulate all of this in the model so it can easily be updated if necessary at a later date and located when necessary. It also makes it easier to test these methods in a model test to verify that your AR works as implemented. 

All of this to say that you may find some popular codebases where they are doing the AR directly in the controller, it really depends on your team and how you all want to organize things.